src/cppcache/integration-test/testThinClientFixedPartitionResolver.cpp

```diff
@@ -245,7 +245,7 @@ DUNIT_TASK_DEFINITION(CLIENT1, CheckPrSingleHopForIntKeysTask_REGION)
         if (networkhop) {
           failureCount++;
         }
-        int8 serverGroupFlag = TestUtils::getCacheImpl(getHelper()->cachePtr)
+        int8_t serverGroupFlag = TestUtils::getCacheImpl(getHelper()->cachePtr)
```

Comment:
I looked into the source, serverGroupFlag is not boolean value but rather an enum.